source/blog/2016-12-06-part1-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,349 @@
+---
+title: 'Installing an OpenShift Origin Cluster on Fedora 25 Atomic Host: Part 1'
+author: dustymabe
+date: 2016-12-06 15:23:32 UTC
+tags: fedora, atomic, origin, openshift, kubernetes
+published: true
+comments: true
+---
+
+Introduction
+============
+
+[Openshift Origin](https://github.com/openshift/origin) is the upstream
+project that builds on top of the Kubernetes platform and feeds into the
+OpenShift Container Platform product that is available from Red Hat today.
+Origin is a great way to get started with Kubernetes and what better
+place to run a container orchestration layer than on top of Fedora
+Atomic Host!
+
+We recently released [Fedora
```

Comment:
Don't like either of these.  Instead: "We recently released Fedora 25, along with the first biweekly release of Fedora 25 Atomic Host."